### PR TITLE
LieAlgebras: Add section with Dynkin diagrams to docs

### PR DESCRIPTION
**Pull request by lgoettgens**
preview: https://docs.oscar-system.org/previews/PR4390/Experimental/LieAlgebras/cartan_matrix/

~~The bib-entry for the Bourbaki is still missing (ping @felix-roehrich).~~

### CODE REVIEW

**Review by felix-roehrich — APPROVED** [review]

File: docs/oscar_references.bib [hunk 1]
```diff
@@ -440,6 +440,16 @@ @Article{Bis96
...
 }
 
+@Book{Bou02,
+  author        = {Bourbaki, Nicolas},
+  title         = {Lie Groups and Lie Algebras: Chapters 4--6},
+  series        = {Elements of Mathematics},
+  note          = {Original French edition published with the title: Éléments de Matematique, Groupes et Algèbres de
+                  Lie 4, 5, e 6, 1968. Translated by Pressley},
+  publisher     = {Springer},
+  year          = {2002}
```
Comment (fingolfin) [inline docs/oscar_references.bib:450]: Entry from mathScinet, including a DOI:
```
@book {MR1890629,
    AUTHOR = {Bourbaki, Nicolas},
     TITLE = {Lie groups and {L}ie algebras. {C}hapters 4--6},
    SERIES = {Elements of Mathematics (Berlin)},
      NOTE = {Translated from the 1968 French original by Andrew Pressley},
 PUBLISHER = {Springer-Verlag, Berlin},
      YEAR = {2002},
     PAGES = {xii+300},
      ISBN = {3-540-42650-7},
   MRCLASS = {17-01 (00A05 20E42 20F55 22-01)},
  MRNUMBER = {1890629},
       DOI = {10.1007/978-3-540-89394-3},
       URL = {https://doi.org/10.1007/978-3-540-89394-3},
}
```

Comment (lgoettgens) [reply docs/oscar_references.bib:450]: The doi unfortunately results in a soft 404 (which somewhat goes against the idea of dois....)

The entry was taken from springer's webpage

File: docs/oscar_references.bib [hunk 2]
```diff
@@ -440,6 +440,16 @@ @Article{Bis96
...
+}
```
Comment (lgoettgens) [inline docs/oscar_references.bib:451]:
```suggestion
@Book{Bou02,
  author        = {Bourbaki, Nicolas},
  title         = {Lie groups and Lie algebras. Chapters 4--6},
  series        = {Elements of Mathematics (Berlin)},
  note          = {Translated from the 1968 French original by Andrew Pressley},
  publisher     = {Springer-Verlag, Berlin},
  year          = {2002},
  doi           = {10.1007/978-3-540-89394-3}
}
```
This is the mathscinet version after applying `bibtool`. If you like this better feel free to apply this change and proceed with merging @fingolfin. I don't care enough to block this PR over it.

Comment (lgoettgens) [reply docs/oscar_references.bib:451]: gentle bump @fingolfin

**Review by fingolfin — APPROVED** [review]

**Pull request merged**